pyproject.toml

```diff
@@ -46,6 +46,7 @@ include = [
   { path = "dev", format = "sdist" },
   { path = "pyiceberg/**/*.so", format = "wheel" },
   { path = "pyiceberg/**/*.pyd", format = "wheel" },
+  { path = "poetry.lock", format = "sdist" },
```

Comment:
We either have to add a ASv2 licence header to the file, or add it to the RAT ignore list.

Reply:
looks like its already added! :) 

https://github.com/apache/iceberg-python/blob/19ba343a2e16823e79f0c0359411f8bfd792ea6e/dev/.rat-excludes#L6

Reply:
Nice, thanks for checking 👍